stingray/bispectra.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+
+from __future__ import division
+import numpy as np
+from scipy.linalg import hankel
+import scipy.fftpack
+import stingray.lightcurve as lightcurve
+import stingray.utils as utils
+from stingray.utils import simon
+__all__ = ["bispectra"]
+
+
+class bispectra(object):
```

Comment:
Yes, I think Powerspectrum should be allowed to be an empty object. I've come across that problem many times in my work where I had something I'd calculated elsewhere that was a power spectrum that I then needed to somehow insert into that class. 

Unless there's a better way to solve that problem?